contentcuration/contentcuration/view/node_views.py

```diff
@@ -15,6 +15,87 @@
 from contentcuration.statistics import record_node_duplication_stats
 
 
+
+def get_node_diff(request):
+
+
+    from datetime import datetime
+    start_time = datetime.now()
+
+
+
+    if request.method == 'POST':
+        channel_id = json.loads(request.body)['channel_id']
+        original = []   # Currently imported nodes
+        changed = []    # Nodes from original node
+        fields_to_check = ['title', 'description', 'license', 'license_description', 'copyright_holder', 'author', 'extra_fields']
+        assessment_fields_to_check = ['type', 'question', 'hints', 'answers', 'order', 'raw_data', 'source_url', 'randomize']
+
+
+        current_tree_id = Channel.objects.get(pk=channel_id).main_tree.tree_id
+        nodes = ContentNode.objects.prefetch_related('assessment_items').prefetch_related('files').prefetch_related('tags')
+
+        copied_nodes = nodes.filter(tree_id=current_tree_id).exclude(original_source_node_id=F('node_id'))
+        channel_ids = copied_nodes.values_list('original_channel_id', flat=True).exclude(original_channel_id=channel_id).distinct()
+        tree_ids = Channel.objects.filter(pk__in=channel_ids).values_list("main_tree__tree_id", flat=True)
+        original_node_ids = copied_nodes.values_list('original_source_node_id', flat=True).distinct()
+        original_nodes = nodes.filter(tree_id__in=tree_ids, node_id__in=original_node_ids)
+
+        # Use dictionary for faster lookup speed
+        content_id_mapping = {n.content_id: n for n in original_nodes}
+
+
+        mid1_time = datetime.now()
+        print "\n\n\n\nQuery Time: {time}s".format(time=(mid1_time - start_time).total_seconds())
```

Comment:
Is this just for benchmarking purposes? Might be good to remove in production code.